test/fixtures/source-map/output/source_map_throw_async_stack_trace.mts

```diff
@@ -3,19 +3,19 @@
 import '../../../common/index.mjs';
 
 interface Foo {
-  /** line
-   *
-   * blocks */
```

Comment:
It seems to me like these kinds of changes are not desirable. I don't think fixtures should be linted.